2020-09-10 16:10:49,054 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 15, error info: name 'yh' is not defined
2020-09-10 16:10:49,056 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 16, error info: name 'yh' is not defined
2020-09-10 16:10:49,057 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: name 'yh' is not defined
2020-09-10 16:10:49,345 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 15, error info: name 'yh' is not defined
2020-09-10 16:10:49,348 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 16, error info: name 'yh' is not defined
2020-09-10 16:10:49,350 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: name 'yh' is not defined
2020-09-10 16:12:41,404 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:12:41,405 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:12:48,882 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:12:48,884 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:12:59,275 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:12:59,276 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:14:13,393 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:14:13,394 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:14:13,700 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:14:13,701 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\structure\\config/normal.yaml'
2020-09-10 16:17:14,597 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\config/normal.yaml'
2020-09-10 16:17:14,598 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\config/normal.yaml'
2020-09-10 16:17:14,817 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\config/normal.yaml'
2020-09-10 16:17:14,819 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: [Errno 2] No such file or directory: 'D:\\GitRepository\\IntelligentSupportPlatform\\indexes\\config/normal.yaml'
2020-09-10 16:17:55,584 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 39, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:17:55,585 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:17:55,586 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:17:56,756 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 39, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:17:56,757 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:17:56,758 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:19:13,986 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 39, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:19:13,987 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:19:13,988 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:21:40,703 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 39, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:21:40,704 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:23:59,092 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:23:59,094 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:23:59,095 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:25:05,195 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:25:05,197 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:25:05,199 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:26:46,626 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:26:46,628 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:26:46,630 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}, range_code={type=keyword}, range_version={type=keyword}, std_name={analyzer=snowball, type=text}, range_type={type=keyword}}}]')
2020-09-10 16:29:21,770 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:21,771 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:21,772 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:26,827 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:26,829 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:26,831 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_name={analyzer=snowball, type=text}}}]')
2020-09-10 16:29:52,896 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 16:29:52,897 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 16:29:52,899 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 16:30:02,089 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 16:30:02,090 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 16:30:02,093 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 17:32:46,575 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\elasticsearch_helper.py, lineon: 40, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 17:32:46,576 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_std_init.py, lineon: 17, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
2020-09-10 17:32:46,577 - log_helper.py[line:60] - ERROR: File: D:\GitRepository\IntelligentSupportPlatform\indexes\structure\medical_range\range_matching_flow.py, lineon: 16, error info: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={std_code={type=keyword}}}]')
